letsencrypt/cli.py

```diff
@@ -1139,6 +1147,26 @@ def main(cli_args=sys.argv[1:]):
 
     return args.func(args, config, plugins)
 
+def check_config_sanity(args):
+    """Validate command line options and display error message if
+    requirements are not met.
+
+    :param args: Command line options
+    :type args: :class:`argparse.Namespace`
+
+    """
+    # Domain checks
+    if args.domains is not None:
+        # Check if there's a wildcard domain
+        if any(True for d in args.domains if d.startswith("*.")):
```

Comment:
that is still not fixed: `any(d.startswith(...) fr d in args.domains)`. Otherwise this espression doesn't make much sense; same below